config/settings.py

```diff
@@ -881,6 +889,8 @@ class CI(EmptyDefaults, Common, use_environ=True):
     # Migrations require the database
     DATABASES = values.DatabaseURLValue()
 
+    FRONTEND_TESTING_API_ENABLED = values.BooleanValue(default=True)
```

Comment:
I would add `FRONTEND_TESTING_API_ENABLED=True` to the `EmptyDefaults` mixin instead.

Reply:
Set this as `True` in `EmptyDefaults` and as `False` in Platta`